### ISSUE TITLE
Window partially hidden when menu bar set to automatically hide (macOS)

### ISSUE THREAD

**Issue by michaelenger** [maxogden/menubar#147]
If you set the menu bar to be automatically hidden on macOS (System Settings > General) the window is positioned under the menu bar.

![screen shot 2016-10-24 at 19 15 47](https://cloud.githubusercontent.com/assets/626587/19656253/7a0b1816-9a1f-11e6-9baf-cc8b21b4dc1f.png)

Not sure if this is something that can be easily fixed by me, or if it should be handled in the menubar library.

**Comment by lucasbento** [comment]
@michaelenger: have you managed to fix this? if so, how?

**Comment by michaelenger** [comment]
@lucasbento No, I haven't. I messed around with it a bit, but didn't find a good solution which is why I opened this issue.

**Comment by IbrahimIrfan** [comment]
@michaelenger Any luck?

**Comment by michaelenger** [comment]
@IbrahimIrfan I haven't even thought about this issue in years, so there is no chance that I'm going to solve it. Sorry 😕

**Comment by IbrahimIrfan** [comment]
If this is still relevant to anyone, this is one option:

```
mb.on("after-show", () => {
    setInterval(() => {
      const trayBounds = mb.tray.getBounds();
      const windowBounds = mb.window.getBounds();

      mb.window.setPosition(
        windowBounds.x,
        Math.max(trayBounds.y + trayBounds.height, 0),
        true
      );
    }, 100);
});
```

A little heavy but can clear the interval on window hide
